spec/rubocop/cop/rspec/string_as_instance_double_constant_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::StringAsInstanceDoubleConstant,
+               :config do
+  context 'when using a class for instance_double' do
+    it 'does not register an offense' do
+      expect_no_offenses(<<~RUBY)
+        instance_double(Foo, failure_message, stubs)
+      RUBY
+    end
+  end
+
+  context 'when using a symbol for instance_double' do
+    it 'does not register an offense' do
+      expect_no_offenses(<<~RUBY)
+        instance_double(:Foo, failure_message, stubs)
+      RUBY
+    end
+  end
```

Comment:
Maybe we can remove this test, if RSpec would raise an ArgumentError for it anyway.